addons/binding/org.openhab.binding.dwdunwetter/README.md

```diff
@@ -0,0 +1,114 @@
+# DwdUnwetter Binding
+
+Binding to retrieve the Weather Warnings of the "Deutscher Wetterdienstes" from the [DWD Geoserver](https://maps.dwd.de/geoserver/web/ "DWD Geoserver").
+The DWD provides weather warnings for the german area. Regions outside of germany are not supported.
+
+## Supported Things
+
+This binding supports exactly one thing - Weather Warning. One Thing provides one or multiple warnings for a city.
```

Comment:
While you're at it... can you also start each sentence on a new line in the lines above the comment?

Reply:
Sure